src/components/AddXpubDialog/AddXpubDialog.tsx

```diff
@@ -75,8 +68,22 @@ export const AddXpubDialog = ({ className }: AddXpubDialogProps) => {
       errorWrapper(error);
     }
   };
+
+  const onGet = () => {
```

Comment:
IMHO this feature that you're just inserting xpriv and it will automatically change into xpub was cooler then the `Get` button.

Maybe we could have still something like that, so 
1. when xpriv changed then automatically calculate xpub 
2. if someone start to enter xpriv, then clear xpub
3. if someone start to type xpub, then clear xpriv

Reply:
Removed `Get` and added auto-convert feature